xml/System.Linq/Enumerable.xml

```diff
@@ -12818,6 +12818,7 @@ In Visual Basic query expression syntax, an `Aggregate Into Sum()` clause transl
         </remarks>
         <exception cref="T:System.ArgumentNullException">
           <paramref name="source" /> is <see langword="null" />.</exception>
+        <exception cref="T:System.OverflowException">The sum is larger than <see cref="F:System.Double.MaxValue">Double.MaxValue</see>.</exception>
```

Comment:
@eiriktsarpalis I added the OverflowExceptions for the double and float overloads. Is this correct?

Reply:
It's not applicable to float overloads, it will return `PositiveInfinity` in those cases.